scss/_sphinx_layout.scss

```diff
@@ -147,10 +147,40 @@
   }
 }
 
+.pytorch-left-nav-collapsible {
+  display: block;
+  background-image: url(../images/chevron-down-orange.svg);
+  background-repeat: no-repeat;
+  background-size: 15px 8px;
+  background-position: 46px 9px;
+}
+
+.pytorch-left-nav-collapsed {
+  display: block;
+  background-image: url(../images/chevron-right-orange.svg);
+  background-repeat: no-repeat;
+  background-size: 15px 11px;
+  background-position: 46px 7px;
+}
+
+.pytorch-left-nav-collapsible:hover {
+  cursor: pointer;
```

Comment:
Instead of giving the entire `p` element `cursor: pointer`, it might be a better idea to limit `cursor: pointer` to the span tag (just the "Notes" text) within `.pytorch-left-nav-collapsible`. With the way it's set up right now you can click all the way to the right of "Notes" and collapse the menu. You'll probably also have to update how you're adding the arrow icons so that they still have `cursor: pointer`.